packages/core/src/account/types.ts

```diff
@@ -136,11 +136,16 @@ export interface ISmartContractAccount<
 
   /**
    * @returns the address of the factory contract for the smart account
+   *
+   * The factory address of the smart account created by the factory
```

Comment:
sorry for this nit... can you move this above the `@returns`? This will be a linting error once I get my jsdocs linter in

Order should be:
- description
- param(s)
- returns